pkg/restapi/v1/oidc4ci/controller.go

```diff
@@ -571,6 +584,10 @@ func (c *Controller) OidcCredential(e echo.Context) error {
 		var interactionErr *interactionError
 
 		if errors.As(parsedErr, &interactionErr) {
+			if interactionErr.Code == resterr.InvalidOrMissingProofOIDCErr {
+				return resterr.NewOIDCError(string(resterr.InvalidOrMissingProofOIDCErr), errors.New(interactionErr.Message))
+			}
```

Comment:
can it be done as another case in switch below ?

Reply:
Yes, good catch! Just updated the PR